tools/xabuild/SymbolicLink.cs

```diff
@@ -21,11 +21,15 @@ public static bool Create (string source, string target)
 					}
 				} else {
 					try {
+						var sourceInfo = new UnixFileInfo (source);
 						var fileInfo = new UnixFileInfo (target);
 						fileInfo.CreateSymbolicLink (source);
-					} catch (Exception exc) {
-						Console.Error.WriteLine ($"Unable to create symbolic link from `{source}` to `{target}`: {exc.Message}");
-						return false;
+					} catch (UnixIOException exc) {
+						if (exc.ErrorCode == Mono.Unix.Native.Errno.EEXIST) {
+							return true;
+						}
+						Console.Error.WriteLine ($"Unable to create symbolic link from `{source}` to `{target}`: {exc}");
+						return Directory.Exists (source);
```

Comment:
I think if we add the same `return Directory.Exists (source)` for Windows, we're good.

I tested what happens on Windows if you try to create a symlink (and it already exists). It gives a generic error message `The system cannot find the file specified`, so we don't have a way to check for the specific error condition.